src/installer/pkg/sfx/Microsoft.NETCore.App/Microsoft.NETCore.App.Runtime.props

```diff
@@ -129,7 +129,7 @@
   <Target Name="AddFrameworkFilesToPackage" DependsOnTargets="ResolveLibrariesFromLocalBuild">
     <ItemGroup>
       <ReferenceCopyLocalPaths Include="@(LibrariesRuntimeFiles)"
-        Condition="'%(LibrariesRuntimeFiles.Extension)' != '.a' or '$(TargetsMobile)' == 'true'">
+        Condition="'%(LibrariesRuntimeFiles.Extension)' != '.a' or '$(TargetsMobile)' == 'true' or '$(BuildNativeAOTRuntimePack)' == 'true'">
```

Comment:
```suggestion
        Condition="'%(LibrariesRuntimeFiles.Extension)' != '.a' or '$(BuildNativeAOTRuntimePack)' == 'true'">
```
?

Reply:
The `'$(TargetsMobile)' == 'true'` part has been there for 2 years so it's needed for something else.